src/main/java/fruitshop/model/FruitTransaction.java

```diff
@@ -0,0 +1,69 @@
+package fruitshop.model;
+
+public class FruitTransaction {
+    private Operation operation;
+    private String fruit;
+    private int quantity;
+
+    public FruitTransaction(Operation operation, String fruit, int quantity) {
+        this.operation = operation;
+        this.fruit = fruit;
+        this.quantity = quantity;
+    }
+
+    public FruitTransaction() {
+    }
+
+    public Operation getOperation() {
+        return operation;
+    }
+
+    public FruitTransaction setOperation(Operation operation) {
+        this.operation = operation;
+        return this;
```

Comment:
Following the encapsulation principle, it is not recommended to return `this` in setters. Setters should return `void` instead.